test/integration/api/odata-entities.js

```diff
@@ -0,0 +1,248 @@
+// Copyright 2023 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const { testService } = require('../setup');
+const testData = require('../../data/xml');
+const { exhaust } = require('../../../lib/worker/worker');
+const { v4: uuid } = require('uuid');
+const { sql } = require('slonik');
+
+describe('api: /datasets/:name.svc', () => {
+  describe('GET /Entities', () => {
+    /* eslint-disable no-await-in-loop*/
+    const createSubmissions = async (user, container, count = 1, skip = 0) => {
+      for (let i = 0; i < count; i += 1) {
+        await user.post('/v1/projects/1/forms/simpleEntity/submissions')
+          .send(testData.instances.simpleEntity.one
+            .replace(/one/g, `submission${i+skip}`)
+            .replace('uuid:12345678-1234-4123-8234-123456789abc', uuid()))
+          .set('Content-Type', 'application/xml')
+          .expect(200);
+
+        await user.patch(`/v1/projects/1/forms/simpleEntity/submissions/submission${i+skip}`)
+          .send({ reviewState: 'approved' })
+          .expect(200);
+
+      }
+      await exhaust(container);
+    };
+    /* eslint-enable no-await-in-loop*/
+
+    it('should return all entities', testService(async (service, container) => {
+      const asAlice = await service.login('alice');
+
+      await asAlice.post('/v1/projects/1/forms?publish=true')
+        .set('Content-Type', 'application/xml')
+        .send(testData.forms.simpleEntity)
+        .expect(200);
+
+      await createSubmissions(asAlice, container, 2);
+
+      await asAlice.get('/v1/projects/1/datasets/people.svc/Entities')
+        .expect(200)
+        .then(({ body }) => {
+          body.value.length.should.be.eql(2);
```

Comment:
I'm not seeing the entity data when I use this endpoint. The values of the properties are blank strings.
Can there be a test that checks that the properties (like first name and age, in this case) are present?